Dockerfile

```diff
@@ -38,7 +38,9 @@ RUN wget --progress=bar:force -P /tmp http://www.neuro.uni-jena.de/cat12/${CAT_F
  && mv /opt/${CAT_FULLVERSION}_${MATLAB_VERSION}_MCR_Linux /opt/spm \
  && /opt/spm/run_spm12.sh ${MCRROOT} --version \
  && chmod +x /opt/spm/spm12 /opt/spm/*.sh \
+ && chmod +x /opt/spm/spm12_mcr/home/gaser/gaser/spm/spm12/toolbox/cat12/CAT.glnx86/CAT_* \
```

Comment:
Funny, those files were executable for all users in older CAT versions

Reply:
They are in the git repo, but it's missing the permission bits after the build.

The call to `/opt/spm/run_spm12.sh ${MCRROOT} --version ` is what's unpacking `spm12.ctf` implicitly, right?

They have all-executable set in the git repo, but `zipinfo` lists the files as non-executable in the current standalone zip.

```
root@981c119aa9aa:~/CAT12.8.1_r2042_R2017b_MCR_Linux# zipinfo spm12.ctf  | grep -e w32/CAT_Surf2Vol -e lnx86/CAT_Surf2Vol
-rw----     0.0 fat  1993150 b- defN 22-Jul-19 00:18 fsroot/home/gaser/gaser/spm/spm12/toolbox/cat12/CAT.glnx86/CAT_Surf2Vol
-rwx---     0.0 fat  1759485 b- defN 22-Jul-19 00:18 fsroot/home/gaser/gaser/spm/spm12/toolbox/cat12/CAT.w32/CAT_Surf2Vol.exe
```